From 9219f420b121129ac1b861a82e9bd4379a082aed Mon Sep 17 00:00:00 2001
From: NgoHuy <huynhok.uit@gmail.com>
Date: Sat, 19 Apr 2014 14:32:52 +0700
Subject: [PATCH 2/3] Remove distro name in License

---
 installer.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/installer.sh b/installer.sh
index edd6a79..1319af5 100755
--- a/installer.sh
+++ b/installer.sh
@@ -16,7 +16,6 @@ LICENSE='Xin chào, đây là bộ cài đặt ibus-ringo, một phần mềm t
 
                      GNU GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
-"Debian GNU/Linux"
  Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
-- 
1.9.2

